src/index.js

```diff
@@ -62,15 +66,12 @@ function bindEvents (options) {
   const eventListenersMap = {
     ready: () => { options.onReady() },
     documentCapture: data => { options.onDocumentCapture(stripOneCapture(data)) },
+    documentBackCapture: data => { options.onDocumentCapture(stripOneCapture(data)) },
     faceCapture: data => { options.onFaceCapture(stripOneCapture(data)) },
     complete: data => { options.onComplete(strip(data)) }
   }
 
-  forEach(eventListenersMap, (listener, event) => {
-    if (event === 'ready') events.once(event, listener)
-    else events.on(event, listener)
-  })
-
+  forEach(eventListenersMap, (listener, event) => events.once(event, listener))
```

Comment:
It will not fire again.

I have an assumption throughout that there will only be one confirmed capture for face, document, reverse document. Is that a bad assumption?